File: day 049/classwork/classwork.py
arr = [
    [False, False, False],
    [True, True, True],
    [False, False, False]
]

for i in range(len(arr)):
    for n in range(len(arr[i])):
        if arr[i][n] == True:
            arr[i][n] = False
        elif arr[i][n] == False:
            arr[i][n] = True
print(arr)

File: day 228/classwork/classwork.js
const array = []
const length = array.length-1
const mid = Math.floor(array.length - 1 / 2)
function binarySearch(arr, target){
    const sortedArr = array.sort()
    while(target === array[mid]){
        if(target === array[mid]){
            return mid
        }
        else if(target < array[mid]){
            length = length - mid
            mid = Math.floor(length / 2)
        }
    }
}